index.html

```diff
@@ -40,13 +51,19 @@ <h1>New Entry</h1>
         </div>
       </div>
 
-      <div class="notes-label"><label for="##">Notes</label></div>
-      <textarea name="notes" class="notes column-full" required="required" /></textarea>
+      <div class="notes-label"><label for="notes-id">Notes</label></div>
```

Comment:
This should just be `for="notes"` so that it matches the `id` of your `textarea`